src/main/java/org/apache/maven/plugins/jira/JiraReport.java

```diff
@@ -333,9 +333,7 @@ public void executeReport(Locale locale) throws MavenReportException {
             if (mockDownloader != null) {
                 issueDownloader = mockDownloader;
             } else {
-                AdaptiveJiraDownloader downloader = new AdaptiveJiraDownloader();
-                downloader.setForceClassic(forceRss);
```

Comment:
`forceRss` parameter can also be removed

Reply:
done